bin/github-backup

```diff
@@ -356,10 +356,9 @@ def _request_http_error(exc, auth, errors):
         print('Exceeded rate limit of {} requests; waiting {} seconds to reset'.format(limit, delta),  # noqa
               file=sys.stderr)
 
-        ratelimit_error = 'No more requests remaining'
```

Comment:
Perhaps I'm missing something... 

Within the semantics of this script, an "error" is anything that should cause the script to quit. If we were to exit `_request_http_error` with a non-empty `errors` list (as was previously the case) the script is **guaranteed** to terminate when it resumes execution from sleeping (see line 285). So my backup will chew for a while, hit the rate limit, go to sleep -- zzz -- wake up, then...die.

So, my point is that hitting the rate limit (whether running as authenticated or not) should _not_ be considered an "error" since we "handle" it by sleeping until the rate limit is reset. At most, we should inform the user that the backup script is going to sleep for a while.